index.html

```diff
@@ -109,78 +113,83 @@ <h4 class="project-client">Canopy</h4>
           </ul>
           <button class="more">see project</button>
         </div>
+        <img
+          src="./image/Snapshoot Portfolio 41.png"
+          alt="multipost"
+          class="project-img"
+        />
       </div>
       <div class="project tonic2">
         <img
-          src="./image/Snapshoot Portfolio 3.svg"
+          src="image\Snapshoot Portfolio 1a.png"
           alt="tonic"
           class="project-img"
         />
-
         <div class="project-details">
-          <h2 class="project-title">Tonic</h2>
+          <h2 class="project-title">Facebook 360</h2>
           <nav class="project-nav">
-            <h4 class="project-client">Canopy</h4>
+            <h4 class="project-client">Facebook</h4>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="pro-lang">Back End Dev</span>
+            <span class="pro-lang">Full stack Dev</span>
             <img src="./image/Counter.svg" alt="counter" />
             <span class="year">2015</span>
           </nav>
           <p class="project-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            Exploring the future of media in 
+            Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
+            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
-          <button class="more">see project</button>
+          <button class=" more more-about">see project</button>
         </div>
       </div>
       <div class="project multipost2-project">
-        <img
-          src="./image/Snapshoot Portfolio4.svg "
-          alt="tonic"
-          class="project-img"
-        />
-
         <div class="project-details">
           <h2 class="project-title">
-            Multipost <br />
-            stories
+            Uber Navigation
           </h2>
           <nav class="project-nav">
-            <h4 class="project-client">Canopy</h4>
+            <h4 class="project-client">Uber</h4>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="pro-lang">Back End Dev</span>
+            <span class="pro-lang">Lead Developer</span>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="year">2015</span>
+            <span class="year">2018</span>
           </nav>
           <p class="project-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A smart assistant to make driving more safe, efficient, and fun by 
```

Comment:
- [ ] [Optional] I would suggest you please use the active, hover, and pressed for your button design.
- [ ] [Optional] Also please remove the background used for buttons here and make sure that when user hover and pressed then only is changes.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213682439-1992da20-2c44-4496-85fc-9c8cb07f710b.jpg)

### Figma design for buttons.

![fimga](https://user-images.githubusercontent.com/81727988/213682988-f2d78b29-c7bb-47ca-854d-05538c298cb3.jpg)